Foodpanda-463/styles.scss

```diff
@@ -0,0 +1,906 @@
+@use './adChoice.scss';
+
+iframe {
+    display: none;
+}
+
+* {
+    box-sizing: border-box;
+    margin: 0;
+    padding: 0;
+}
+
+.LM_container {
+    width: 100vw;
+    height: 100vh;
+    position: relative;
+    overflow: hidden;
+}
+
+@mixin fullScreen {
+    width: 100%;
+    height: 100%
+ }
+
+.LM_loading_page {
+    display: none;
+    position: absolute;
+    z-index: 100000;
+    background: #000;
+    @include fullScreen;
+
+    .preloader {
+        display: inline-block;
+        font-size: 0;
+        padding: 0;
+        position: absolute;
+        top: 50%;
+        left: 50%;
+    }
+}
+
+.LM_landscape {
+    display: none;
+    position: absolute;
+    top: 0;
+    left: 0;
+    bottom: 0;
+    right: 0;
+    background: #ffffff url(https://i.loopme.me/html/general/not_available.png) no-repeat;
+    background-position: center center;
+    background-size: 70px;
+    z-index: 100000000;
+}
+
+.LM_content {
+    @include fullScreen;
+    overflow: hidden;
+    background: rgb(255, 255, 255);
+    display: flex;
+    flex-direction: column;
+    justify-content: center;
+
+    .logoZone {
+        height: 10%;
+        position: relative;
+        display: flex;
+        flex-direction: column;
+        justify-content: center;
+        align-items: center;
```

Comment:
If you use these same scss properties multiple times, you can replace it with mixin, and if you have the same code used twice, you can use mixin